artifactory/utils/dotnet/solution/solution.go

```diff
@@ -80,7 +80,11 @@ func (solution *solution) BuildInfo(moduleName string) (*buildinfo.BuildInfo, er
 
 		// Populate module dependencies
 		for _, dep := range dependencies {
-			module.Dependencies = append(module.Dependencies, *dep)
+			// If dependnecy has no RequestedBy field, it is not accessible in the current project -
```

Comment:
If dependnecy has no RequestedBy field, it is not accessible in the current project -
-->
If dependency has no RequestedBy field, it means that the depedency not accessible in the current project.